.github/workflows/commit-check-pr.yml

```diff
@@ -62,15 +62,15 @@ jobs:
     needs: fetch-data
     env:
       PR_NUMBER: ${{ needs.fetch-data.outputs.pr_number }}
+      PR_HASH: ${{ needs.fetch-data.outputs.head_sha }}
     steps:
       - uses: actions/checkout@v3
       - name: Get Commits
```

Comment:
I would suggest adding a step to include `fetch-depth: 0` here, to fetch all history for all branches and tags.

Reply:
or a  `git fetch --all`

Reply:
I don't think we need all branch history for this, we're just comparing main branch with the pr sha commit